### PR TITLE
  	new file:   Casks/liclipse.rb

### PR DESCRIPTION
**Pull request by tjt263**
LiClipse 2.1.0

### CODE REVIEW

File: Casks/liclipse.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'liclipse' do
+  version '2.1.0'
+  sha256 '3b6077e3208d92ac313f685f05fd20aee6705df9461db88a9141899969a7b623'
+
+  url 'https://googledrive.com/host/0BwwQN8QrgsRpLVlDeHRNemw3S1E/LiClipse%202.1.0/liclipse_2.1.0_macosx.cocoa.x86_64.dmg'
+  name 'LiClipse'
+  homepage 'http://www.liclipse.com/'
+  license :other
+  
+  suite 'liclipse', :target => 'LiClipse'
```
Comment (vitorgalvao) [inline Casks/liclipse.rb]: Please [do not `:target` for no reason](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#target-should-only-be-used-in-select-cases).

File: Casks/liclipse.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  app 'liclipse/LiClipse.app'
+  binary 'liclipse/LiClipse.app/Contents/MacOS/LiClipse'
```
Comment (vitorgalvao) [inline Casks/liclipse.rb]: Don’t do this unless it is officially recommended. We won’t start adding binaries without users expecting them.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 75b85b4e9f4751021dc78d6d2f7124240c4b5ed1. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/liclipse.rb (hunk 2 above)
Comment (tjt263) [inline Casks/liclipse.rb]: ok, im new at this, i was under the impression that would make it quicker and easier to execute via bash shell environment?

Comment (tjt263) [inline Casks/liclipse.rb]: another question; was i right to use `suite' or should i have just used  `app' ?

File: Casks/liclipse.rb (hunk 1 above)
Comment (tjt263) [inline Casks/liclipse.rb]: ok, it was just for aesthetic purposes, to match the style of ~/Applications and LiClipse.app

File: Casks/liclipse.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'liclipse' do
+  version '2.1.0'
+  sha256 '3b6077e3208d92ac313f685f05fd20aee6705df9461db88a9141899969a7b623'
+
+  url 'https://googledrive.com/host/0BwwQN8QrgsRpLVlDeHRNemw3S1E/LiClipse%202.1.0/liclipse_2.1.0_macosx.cocoa.x86_64.dmg'
+  name 'LiClipse'
+  homepage 'http://www.liclipse.com/'
+  license :other
+
```
Comment (tjt263) [inline Casks/liclipse.rb]: i had an error message about trailing whitespace on line#9. whats that about? should i not have skipped a line?
thanks

File: Casks/liclipse.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/liclipse.rb]: > i was under the impression that would make it quicker and easier to execute via bash shell environment?

It does, but we shouldn’t be doing that unexpectedly. Apps should be as close as possible to a vanilla install.

> i right to use `suite' or should i have just used  `app' ?

See the fixed commit. Either one or the other. Both makes no sense.

File: Casks/liclipse.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/liclipse.rb]: You had spaces on that line. It should be empty.

File: Casks/liclipse.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/liclipse.rb]: That’s exactly what the link I pointed you to shows. Your aesthetic changes will not match other people’s.

File: Casks/liclipse.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'liclipse' do
+  version '2.1.0'
+  sha256 '3b6077e3208d92ac313f685f05fd20aee6705df9461db88a9141899969a7b623'
+
+  url 'https://googledrive.com/host/0BwwQN8QrgsRpLVlDeHRNemw3S1E/LiClipse%202.1.0/liclipse_2.1.0_macosx.cocoa.x86_64.dmg'
```
Comment (vitorgalvao) [inline Casks/liclipse.rb]: You were also missing the vendor comment.

**Pull request closed without merging**